### PR TITLE
93 update and organize makefile

### PR DESCRIPTION
**Pull request by forbidden-arts**
Do debug branch first

added an empty.html for siege

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,38 +20,42 @@ CUT			=	\033[K
...
-				FileHandler \
-				ConfigParser \
-				Client \
-				Connection \
-				ServerHandler
+				Methods \
+				Server \
+				ServerHandler \
+				Socket \
+				Util
```
Comment (merituulie) [inline Makefile:38]: Im surprised by the order. I assume that the parts that are used in other places most like Util, Exceptions etc need to be the first ones to get compiled and the ones that use a lot of other classes like requestparser and serverhandler are the last ones to get compiled.

File: Makefile [hunk 2]
```diff
@@ -20,38 +20,42 @@ CUT			=	\033[K
...
 
-H_FILES =	Exceptions \
-			Util \
+H_FILES =		
 			CgiHandler \
+			Client \
+			ConfigParser \
+			Connection \
+			ExceptionManager \
+			Exceptions \
```
Comment (merituulie) [inline Makefile:46]: Like here, ExceptionManager uses Exceptions so Exceptions should be checked and compiled first and ExceptionManager after that.

Comment (forbidden-arts) [reply Makefile:46]: Alphabetical so I didn't miss anything

Comment (merituulie) [reply Makefile:46]: did it compile?

**Pull request merged**